crates/exex/src/model.rs

```diff
@@ -360,6 +418,22 @@ mod tests {
             // Assert roundtrip conversion is equal to original value
             prop_assert_eq!(roundtrip_address, address);
         }
+
+        #[test]
+        fn test_bloom_to_keth_maybe_relocatable_roundtrip(bloom in any::<Bloom>()) {
+            // Convert Bloom to KethMaybeRelocatables
+            let keth_maybe_relocatables = KethMaybeRelocatables::from(bloom);
+
+            // Convert back to Bloom
+            let roundtrip_bloom = KethMaybeRelocatables::to_bloom(&keth_maybe_relocatables);
+
+            // Assert roundtrip conversion is equal to original value
+            prop_assert_eq!(roundtrip_bloom, bloom);
+            prop_assert_eq!(
+                bloom.len(),
+                keth_maybe_relocatables.0[0].0.get_int().unwrap().to_string().parse::<usize>().unwrap()
```

Comment:
isn't there any easier way to go from a felt to a `usize`?

Reply:
Not from what I know but we could add some helpers in this repo https://github.com/starknet-io/types-rs/blob/main/crates/starknet-types-core/src/felt/mod.rs

Reply:
I don't see them neither, but I think indeed that it's worth implementing the `TryInto<uX> for Felt`

Reply:
Yes will do that